bindings/matrix-sdk-ffi/src/timeline.rs

```diff
@@ -775,6 +782,34 @@ impl TryFrom<&AudioInfo> for BaseAudioInfo {
     }
 }
 
+#[derive(Clone, uniffi::Record)]
+pub struct AudioDetailsContent {
```

Comment:
Can you please carry over the `unstable` naming into the FFI crate?

```suggestion
pub struct UnstableAudioDetailsContent {
```

Reply:
OK, that's done.